CONTRIBUTING.md

```diff
@@ -66,32 +68,51 @@ Other notes:
+of how these are written refer to the current [ADRs](https://github.com/cosmos/cosmos-sdk/tree/master/docs/architecture).
 
 ## Pull Requests
 
-To accommodate review process we suggest that PRs are categorically broken up.
-Ideally each PR addresses only a single issue. Additionally, as much as possible
-code refactoring and cleanup should be submitted as a separate PRs from bugfixes/feature-additions.
+PRs should be categorically broken up based on the type of changes being made (i.e. `fix`, `feat`,
+`refactor`, `docs`, etc.). The *type* must be included in the PR title as a prefix (e.g.
+`fix: <description>`). This ensures that all changes committed to the base branch follow the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification. 
+Additionally, each PR should only address a single issue.
 
-### Process for reviewing PRs
+### Pull Request Templates
 
-All PRs require two Reviews before merge (except docs changes, or variable name-changes which only require one). When reviewing PRs please use the following review explanations:
+There are currently three PR templates. The [default template](./.github/PULL_REQUEST_TEMPLATE.md) is for types `fix`, `feat`, and `refactor`. We also have a [docs template](./.github/PULL_REQUEST_TEMPLATE/docs.md) for documentation changes and an [other template](./.github/PULL_REQUEST_TEMPLATE/other.md) for changes that do not affect production code. When previewing a PR before it has been opened, you can change the template by adding one of the following parameters to the url:
+- `template=docs.md`
+- `template=other.md`
 
-- `LGTM` without an explicit approval means that the changes look good, but you haven't pulled down the code, run tests locally and thoroughly reviewed it.
-- `Approval` through the GH UI means that you understand the code, documentation/spec is updated in the right places, you have pulled down and tested the code locally. In addition:
+### Requesting Reviews
+
+In order to accomodate the review process, the author of the PR must complete the author checklist
+to the best of their abilities before marking the PR as "Ready for Review". If you would like to
+receive early feedback on the PR, open the PR as a "Draft" and leave a comment in the PR indicating
+that you would like early feedback and tagging whoever you would like to receive feedback from.
+
+### Reviewing Pull Requests
+
+All PRs require at least two reviews before they can be merged (one review might be acceptable in
+the case of minor changes to [docs](./.github/PULL_REQUEST_TEMPLATE/docs.md) or [other](./.github/PULL_REQUEST_TEMPLATE/other.md) changes that do not affect production code). Each PR template has a
+reviewers checklist that must be completed before the PR can be merged. Each reviewer is responsible
+for all checked items unless they have indicated otherwise by leaving their handle next to specific
+items. In addition, please use the following review explanations:
```

Comment:
There is discussion about improving this process in the future. https://github.com/cosmos/cosmos-sdk/pull/9342#discussion_r641581888